src/balancerd/tests/server.rs

```diff
@@ -88,15 +88,14 @@ use mz_frontegg_auth::{
 };
 use mz_ore::metrics::MetricsRegistry;
 use mz_ore::now::SYSTEM_TIME;
-use mz_ore::task::RuntimeExt;
+use mz_ore::task;
 use mz_server_core::TlsCertConfig;
 use openssl::ssl::{SslConnectorBuilder, SslVerifyMode};
-use postgres::Client;
 use uuid::Uuid;
 
-#[mz_ore::test]
+#[mz_ore::test(tokio::test(flavor = "multi_thread", worker_threads = 1))]
```

Comment:
What happens if you leave out `flavor = "multi_thread", worker_threads = 1`?

Reply:
For some reason the test hangs, everything runs but we're blocked on cleanup. I haven't figured out exactly what's happening yet, but will dig in further when I have a free cycle or two